README.md

```diff
@@ -15,9 +15,9 @@
 
 ### Learn how to design large-scale systems
 
-Learning how to design scalable systems will help you become a better engineer.
+Being able to design scalable systems will help you become a better engineer.
 
-System design is a broad topic.  There is a **vast amount of resources scattered throughout the web** on system design principles.
+System design is a broad topic and there are a **vast amount of resources scattered throughout the web** on system design principles.
```

Comment:
I think we should remove `a` as well. It should be `System design is a broad topic and there are vast amount of resources....`